arm-service-map/2015-11-01-preview/swagger/arm-service-map.json

```diff
@@ -1299,40 +1215,235 @@
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      },
+      "post": {
+        "tags": [
+          "MachineGroups"
+        ],
+        "operationId": "MachineGroups_Create",
+        "description": "Creates a new machine group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "machineGroup",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MachineGroup"
+            },
+            "description": "Machine Group resource to create."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/MachineGroup"
+            }
+          },
+          "default": {
+            "description": "An error occurred while processing the request. See the error.code parameter to identify the specific error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/features/serviceMap/machineGroups/{machineGroupName}": {
+      "get": {
+        "tags": [
+          "MachineGroups"
+        ],
+        "operationId": "MachineGroups_Get",
+        "description": "Returns the specified machine group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/MachineGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/MachineGroup"
+            }
+          },
+          "default": {
+            "description": "An error occurred while processing the request. See the error.code parameter to identify the specific error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "MachineGroups"
+        ],
+        "operationId": "MachineGroups_Update",
```

Comment:
The latest linter rules (which you may not have had access to, the latest nightly builds have them) indicates that `PUT` `operationsId`'s should have their verb start with the word `Create`

Is `MachineGroups_Update` ONLY an update, or is it really `CreateOrUpdate` ?